docs/great-houses/house-of-daos.mdx

```diff
@@ -1,104 +1,119 @@
-[What, Why & How of the DAO](https://wiki.metagame.wtf/docs/Guilding/what-why-how-of-your-dao) - Essential questions you should be asking yourself when starting a DAO.
-[Guilding 101](https://wiki.metagame.wtf/docs/Guilding/guilding-101) - How to start DAOs that have a specific goal.
-[Building Successful Teams](https://wiki.metagame.wtf/docs/Guilding/building-successful-teams) - Essentials of team-building.
-[Connection/Community Building Resources](https://forum.metagame.wtf/t/connection-community-building-resources-practices/256)
-Got more useful links? Add them here.
+There are multiple types of “decentralizations”. Some will say “A DAO” only implies architectural decentralization; the technology itself. While one may _have_ “a DAO” whose governance is centralized in a single or a few big shareholders, we do not consider those true DAOs.
 
-### Books
+If you ask most of us, a DAO should be both architecturally AND politically decentralized to truly be considered a DAO. For a DAO to be considered politically decentralized, it’s voting power should be _sufficiently_ decentralized among its members. How much is sufficient? Well who the hell knows, we’ll figure it out.
 
-[The Starfish and the Spider](https://www.goodreads.com/book/show/21314.The_Starfish_and_the_Spider): The Unstoppable Power of Leaderless Organizations
-[Reinventing Organizations](https://www.goodreads.com/book/show/20787425-reinventing-organizations?from_search=true&from_srp=true&qid=9kHh66iPPT&rank=1): A Guide to Creating Organizations Inspired by the Next Stage of Human Consciousness
-[Brave New Work](https://www.goodreads.com/book/show/40338432-brave-new-work?from_search=true&from_srp=true&qid=9kHh66iPPT&rank=2): Are You Ready To Reinvent Your Organization?
-[The Penguin and the Leviathan](https://www.goodreads.com/book/show/7286895-the-penguin-and-the-leviathan): The Triumph of Cooperation Over Self-Interest
-[The Culture Code](https://www.goodreads.com/book/show/33517721-the-culture-code): The Secrets of Highly Successful Groups
+If you ask me, DAOs are here to enable us to build a world of cooperatively ran organizations and collectively owned platforms & infrastructure.
 
-If you have any questions or need any help, you are welcome on our [Discord](https://discord.gg/6JFXC9T) :)
+### To recap:
+
+-   There is no centralized control in the DAO. There is no “CEO” and no single individual may dictate direction.
+-   There isn’t a hierarchy in any traditional sense, no “managerial” & “executive” layers.
+-   Members allocate resources and make decisions about the direction and initiatives through voting on proposals or practicing other decentralized decision making methods, eg. [Holacracy’s Integrative Decision Making Process](https://medium.com/org-hacking/holacracys-integrative-decision-making-process-f750d4b82abc)
+-   There is a complete transparency of every transaction.
```

Comment:
@peth-yursick you need to add a new line just after `There is a complete transparency of every transaction.` and before the `</details>` tag and it should fix the error on the build. It thinks it's the same line and so is putting the closing li after the </details> tag.

Reply:
When transitioning from a list or something in markdown, you need an empty line at the end of the list before using another element, or in this case the `</details>` tag, so the processor knows it's terminated.